packages/create-llama/templates/components/vectordbs/python/none/constants.py

```diff
@@ -1 +1,2 @@
 STORAGE_DIR = "storage"  # directory to cache the generated index
+DATA_DIR = "data"  # directory to store the data
```

Comment:
Why do we need it? Isn't it just used in the loader?

Reply:
Ah yes, then the other vector db should have the DATA_DIR too. I also think file loader can use the DATA_DIR and STORAGE_DIR from environment variable instead, then we can remove the constants.py for none vectordb.